### DISCUSSION TITLE
Generation trip: Generator desconection vs PV gen desconection, RenGen trip doesn't work

### DISCUSSION THREAD

**Discussion by amparocorvalan** [CURENT/andes#558] [Q&A] [answered]
Hello everyone,

I want to study the frequency stability for different renewable penetration levels of the ieee14 system. That is why I need to simulate generation or load trip. I used the ieee14_gentrip file and simulating the disconnection of generator 2 the output of the TDS is that the system is not stable and the following curve is obtained (system2):

![image](https://github.com/user-attachments/assets/190b187a-2263-46bf-a4b7-834e3e43cf8a)

Then, I tried to do the same but instead of disconnecting the generator, I disconnected the PV generation and I got a very different behavior (if generation is disconnected it should decrease the frequency not increase) (systrem1):

![image](https://github.com/user-attachments/assets/e82a86fd-0a77-4a08-b8b0-964e1fae6047)

here are the perturbations simulated:
![image](https://github.com/user-attachments/assets/080e631b-9709-483c-a2d6-da3280023519)

Could someone explain me what is happening. Why doesn't it work the same way ? I tried this other way because when I tried to simulate the tripping of a renewable generation unit nothing happened. The frequency stays at 1 pu. How can I simulate generation trip for renewable units?

![image](https://github.com/user-attachments/assets/da0f44a7-d498-4ea3-b140-6b36ef8ee991)

I would be very grateful if someone could help me.

Regards,
Amparo

**Comment by cuihantao** [comment]
Hi Amparo,

Which renewable generator are you using? Probably it's not coded yet to take status changes. 

Also, tripping a conventional generator and a renewable generators will have the same result, if they are initialized at the same initial power flow.

**Reply by amparocorvalan** [reply]
Hi @cuihantao,

Thank you for your reply! I'm using REGCA1, the model of solar used in ieee14_solar. How so the tripping of a conventional unit has the same result as the tripping of a renewable generator? 

What I want to see is the frequency response under load drop or generation conditions, for systems with high RES penetration. If the renewable generators are not coded yet to take status changes, how could I simulate this generation loss?

Thank you in advance!

Regards,
Amparo

**Comment by cuihantao** [comment, marked as answer]
> I'm using REGCA1, the model of solar used in ieee14_solar. How so the tripping of a conventional unit have the same result as the tripping of a renewable generator?

Consider two scenarios where you connect a conventional generator and REGC_1, respectively. If the two generators are initialized from the same static PV, disconnecting the generator in either case will result in the same instantaneous reduction of power, so the response is expected to be the same. 

> If the renewable generators are not coded yet to take status changes, how could I simulate this generation loss?

You can try editing the power output equation in the renewable generator by multiplying `u`. When `u` goes to zero, the power injection should then reduce to zero.

**Reply by amparocorvalan** [reply]
Thank you! It worked doing that.
